react/react1/week3/week3homework/src/Components/TodoList.js

```diff
@@ -0,0 +1,108 @@
+import { useState, useEffect } from "react";
+import FancyBorder from "./FancyBorder";
+
+function TodoList({ todos, onDelete, onUpdate }) {
+  const [startEditable, setStartEditable] = useState(false);
+  //   const [endEditable, setEndEditable] = useState(false);
+  const [inputValue, setInputValue] = useState(" ");
```

Comment:
This component also has too much responsibility.
A TodoItem component should have the  inputValue state